pnpm-lock.yaml

```diff
@@ -1,253 +1,344 @@
-lockfileVersion: 5.4
+lockfileVersion: '6.0'
```

Comment:
you installed with pnpm 8. This repository uses pnpm 7. Please do:
```sh
pnpm dlx pnpm@^7 install
# or 
npx pnpm@^7 install
```

Reply:
Fixed it